src/Client.php

```diff
@@ -457,6 +447,28 @@ public function watch(array $pipeline = [], array $options = [])
         return $operation->execute($server);
     }
 
+    private function formatEncryptionOptions(array $options): array
```

Comment:
```suggestion
    private function prepareAutoEncryptionOptions(array $options): array
```

I'm not sure if there's prior art in PHPLIB, but I use "prepare" extensively in PHPC for this sort of thing. Up to you, though.

I do like including "AutoEncryption" in the name here since it refers to that driver option.

Reply:
Ok for "prepare", but no for "AutoEncryption": the method `createClientEncryption` isn't specific to auto encryption.